lib/ecto/adapters/mysql.ex

```diff
@@ -29,6 +29,7 @@ defmodule Ecto.Adapters.MySQL do
     * `:ssl` - Set to true if ssl should be used (default: false);
     * `:ssl_opts` - A list of ssl options, see ssl docs;
     * `:connect_timeout` - The timeout for establishing new connections, defaults to `5000`
+    * `:storage_engine` - The storage engine to be used in the current session, i.e: "MYISAM". The default is "INNODB"
```

Comment:
Does it makes sense to have this option for storage_up? Even if the user sets it to MyISAM, the migrations will default to InnoDB, having no effect. I think we can :scissors: this. What do you think?